### PR TITLE
changed chat link to discord

### PR DESCRIPTION
**Pull request by Johan511**
Changing support.md to add discord link

### CODE REVIEW

File: .github/SUPPORT.md [hunk 1]
```diff
@@ -21,16 +21,11 @@ Please send us an [email](info@stellar-group.org) describing the issue. See our
...
 
-* #ste||ar on [Libera.Chat](https://libera.chat/): this is the best option
+* #ste||ar on [Discord](https://discord.gg/Tn9QuzVjvy): this is the best option
   to get in contact with us as most of the developers are available on this
-  channel. Check out the [channel archives](http://irclog.cct.lsu.edu/ste~b~~b~ar)
-  for past conversations. The channel can also be accessed through
-  [Matrix](https://matrix.to/#/#ste||ar:libera.chat).
-* Slack ([registration](https://cpplang.now.sh/)): The
-  [`#hpx`](https://cpplang.slack.com/messages/C68QLPZB3) channel is another
-  place where people usually offer support.
```
Comment (hkaiser) [inline .github/SUPPORT.md:33]: Could you leave the link to the Slack channel in here, please?

Comment (Johan511) [reply .github/SUPPORT.md:33]: Done

**Review by hkaiser — APPROVED** [review]
LGTM, thanks!

**Comment by hkaiser** [comment]
bors merge

**Pull request merged**